Core/Core/Extensions/NSPersistentContainerExtensions.swift

```diff
@@ -18,6 +18,8 @@ import Foundation
 import CoreData
 
 extension NSPersistentContainer {
+    public static let shared = create(appGroup: "group.com.instructure.icanvas")
```

Comment:
should this omit the `appGroup` parameter? The default should be right for student app & extensions. I'd rather not have this group id as a literal in many places.